js/spaceShip.js

```diff
@@ -1,33 +1,62 @@
-var SpaceShip = function(x, y){
+
+
+  //default parameters are to put the spaceShip in the center
+var SpaceShip = function (x = width / 2, y = height / 2) {
```

Comment:
Default values are again ES6 and not supported on some browsers, you can use an if condition in the function to set default values instead